configure.py

```diff
@@ -803,7 +803,7 @@
 parser.add_argument('--v8-enable-object-print',
     action='store_true',
     dest='v8_enable_object_print',
-    default=True,
+    default=False,
```

Comment:
This changes the default configuration (object printing being enabled) which may be undesirable? The logic I outlined previously would maintain the current default while ensuring `--v8-disable-object-print` does what it should.